include/mtx/requests.hpp

```diff
@@ -84,6 +85,21 @@ struct Login
 void
 to_json(json &obj, const Login &request);
 
+//! Request payload for the `POST /_matrix/client/r0/sendToDevice/{eventType}/{transcationID}`
```

Comment:
Minor nit on the comment, from the spec, it appears to be a PUT request rather than a POST request:
https://matrix.org/docs/spec/client_server/latest#put-matrix-client-r0-sendtodevice-eventtype-txnid

Reply:
My Bad I'll Change it